src/cycax/cycad/cycad_side.py

```diff
@@ -578,3 +687,20 @@ def _box_size_calc(self, width: float, length: float, depth: float) -> tuple[flo
 
     def _rotate(self):
         self._parent.rotate_freeze_front()
+
+    def cylinder(
```

Comment:
This is nice. But I think we will run out of words later on. 
We should look at doing:
```
side.cut().cylinder() # Synonym for side.hole()
side.add().cylinder() # Synonym for side.cylinder() 
```

That way when we get box, cuboid, sphere, cone, torus etc. it is the same few primitives but with cut and add. Note we should pick carefully so that 2D and 3D construction have the same flow. Shortcuts like `hole()` is fine since they are so common, and exactly that a shortcut.